### PR TITLE
Playground exclude unavailable packages

### PR DESCRIPTION
**Pull request by whitphx**
## Description

LLM can generate an app that imports Python standard libraries and the LLM-based requirement list generator can add even these package names, which causes the following error.

> PythonError: Traceback (most recent call last):
  File "/lib/python3.12/site-packages/micropip/_commands/install.py", line 146, in install
    raise ValueError(
ValueError: Can't find a pure Python 3 wheel for: 'random'
See: https://pyodide.org/en/stable/usage/faq.html#why-can-t-micropip-find-a-pure-python-wheel-for-a-package

This PR fixes it by checking if the package exists on PyPI and removing unavailable ones from the generated requirements.

Example:
https://87fa2dfb.gradio-website.pages.dev/playground?demo=Hello_World&code=IyBTYW1wbGUgR3JhZGlvIGFwcCB1c2luZyB0aGUgcmFuZG9tIHBhY2thZ2UKaW1wb3J0IGdyYWRpbyBhcyBncgppbXBvcnQgcmFuZG9tCgojIERlZmluZSBhIGZ1bmN0aW9uIHRoYXQgZ2VuZXJhdGVzIGEgcmFuZG9tIGdyZWV0aW5nCmRlZiByYW5kb21fZ3JlZXQobmFtZSk6CiAgICBncmVldGluZ3MgPSBbIkhlbGxvIiwgIkhpIiwgIkhleSIsICJHcmVldGluZ3MiXQogICAgcmV0dXJuIGYie3JhbmRvbS5jaG9pY2UoZ3JlZXRpbmdzKX0ge25hbWV9ISIKCiMgQ3JlYXRlIGEgR3JhZGlvIGludGVyZmFjZSB0aGF0IHRha2VzIGEgdGV4dGJveCBpbnB1dCwgcnVucyBpdCB0aHJvdWdoIHRoZSByYW5kb21fZ3JlZXQgZnVuY3Rpb24sIGFuZCByZXR1cm5zIG91dHB1dCB0byBhIHRleHRib3guCmRlbW8gPSBnci5JbnRlcmZhY2UoZm49cmFuZG9tX2dyZWV0LCBpbnB1dHM9InRleHRib3giLCBvdXRwdXRzPSJ0ZXh0Ym94IikKCiMgTGF1bmNoIHRoZSBpbnRlcmZhY2UuCmlmIF9fbmFtZV9fID09ICJfX21haW5fXyI6CiAgICBkZW1vLmxhdW5jaChzaG93X2Vycm9yPVRydWUp&reqs=

The prompt that generated the example app above:
> Sample app using the random package

### CODE REVIEW

File: js/_website/src/lib/components/DemosLite/DemosLite.svelte [hunk 1]
```diff
@@ -13,14 +13,15 @@
...
 
 	let generated = true;
 
 	let current_code = false;
 	let compare = false;
 
-	const workerUrl = "https://playground-worker.pages.dev/api/generate";
-	// const workerUrl = "http://localhost:5174/api/generate";
+	// const workerUrl = "https://playground-worker.pages.dev/api/generate";
+	const workerUrl = "http://localhost:5174/api/generate";
```
Comment (aliabd) [inline js/_website/src/lib/components/DemosLite/DemosLite.svelte:24]: you forgot to switch them back

Comment (aliabd) [inline js/_website/src/lib/components/DemosLite/DemosLite.svelte:24]:
```suggestion
const workerUrl = "https://playground-worker.pages.dev/api/generate";
// const workerUrl = "http://localhost:5174/api/generate";
```

**Review by aliabd — COMMENTED** [review]
Looks great @whitphx! just need to fix the workerurl

File: js/_website/src/lib/components/DemosLite/DemosLite.svelte (hunk 1 above)
Comment (whitphx) [reply js/_website/src/lib/components/DemosLite/DemosLite.svelte:24]: Oops

Comment (whitphx) [reply js/_website/src/lib/components/DemosLite/DemosLite.svelte:24]: Thanks!

**Comment by whitphx** [comment]
@aliabd Thanks, I updated the PR.

**Review by aliabd — APPROVED** [review]
Great everything lgtm! thanks @whitphx

**Pull request merged**